src/app/shared/utils/error.utils.ts

```diff
@@ -15,10 +15,21 @@ export class ErrorUtils {
       'Transaction failed. This is probably our fault. Contact our support with code: -32700',
   };
 
-  public static parseError(errorCode: EErrorTypes): string {
-    if (!(errorCode in this.JSON_RPC_ERROR_TYPES)) {
-      return 'Something went wrong. The transaction has been cancelled.';
+  public static parseError(errorCode: EErrorTypes, message?: string): string {
+    if (message) {
+      const maxRatio = message.indexOf('ERR_MAX_IN_RATIO') > -1;
+      const minRatio = message.indexOf('ERR_MIN_IN_RATIO') > -1;
+      if (maxRatio || minRatio) {
+        return `Investment value(s) of Balancer pool position(s) in basket are ${maxRatio ? 'too big' : 'too small'}.`;
+      }
     }
-    return this.JSON_RPC_ERROR_TYPES[errorCode];
+    if (!!this.JSON_RPC_ERROR_TYPES[errorCode]) {
+      return this.JSON_RPC_ERROR_TYPES[errorCode];
+    }
+    if (errorCode === EErrorTypes.BASKET_POSITION_INVEST_ERROR) {
```

Comment:
maybe we can add this as a key in JSON_RPC_ERROR_TYPES and handle it with the line on line 26